src/rtapi/rtapi_uspace.hh

```diff
@@ -17,14 +17,21 @@
  */
 #ifndef RTAPI_USPACE_HH
 #define RTAPI_USPACE_HH
+#ifdef __linux__
 #include <sys/fsuid.h>
+#endif
 #include <unistd.h>
 #include <pthread.h>
 
 struct WithRoot
 {
+#ifdef __linux__
     WithRoot() { if(!level) setfsuid(geteuid()); level++; }
     ~WithRoot() { --level; if(!level) setfsuid(getuid()); }
+#else
+    WithRoot() { if(!level) level++; }
```

Comment:
hm, this doesn't seem right.  WithRoot::WithRoot() needs to elevate file I/O privs to root, so it can't simply do nothing.  But in the long run I'd like to get this fixed up so we can abandon use of linux-specific get/setfsuid altogether.